### PR TITLE
[DDW-1064]: Implement connecting and syncing screens for incentivized testnet

### PR DESCRIPTION
**Pull request by yakovkaravelov**
This PR implemented syncing and connecting screens for incentivized testnet
[YouTrack Ticket](https://iohk.myjetbrains.com/youtrack/issue/DDW-1064)

## Todos
- [x] Implement syncing and connecting screens for incentivized testnet

## Screenshots
image
image
image
image

## Testing Checklist
- [Slack QA thread](https://input-output-rnd.slack.com/archives/GGKFXSKC6/p1573528797444200)
- [x] Run app and check if syncing and connecting screens are matching with wireframes
- [x] Go to `Loading` => `SyncingConnecting - Connectivity Issues For Incentivized Testnet` story on storybook and check if it matches with wireframe
- [x] Go to `Loading` => `SyncingConnecting - Sync Issues For Incentivized Testnet` story on storybook and check if it matches with wireframe

### Basics

- [ ] PR has been assigned and has appropriate labels (`feature`/`bug`/`chore`, `release-x.x.x`)
- [ ] PR is updated to the most recent version of the target branch (and there are no conflicts)
- [ ] PR has a good description that summarizes all changes and shows some screenshots or animated GIFs of important UI changes
- [ ] CHANGELOG entry has been added to the top of the appropriate section (*Features*, *Fixes*, *Chores*) and is linked to the correct PR on GitHub
- [ ] Automated tests: All acceptance and unit tests are passing (`yarn test`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *development* build (`yarn dev`)
- [ ] Manual tests (minimum tests should cover newly added feature/fix): App works correctly in *production* build (`yarn package` / CI builds)
- [ ] There are no *flow* errors or warnings (`yarn flow:test`)
- [ ] There are no *lint* errors or warnings (`yarn lint`)
- [ ] There are no *prettier* errors or warnings (`yarn prettier:check`)
- [ ] There are no missing translations (running `yarn manage:translations` produces no changes)
- [ ] Text changes are proofread and approved (Jane Wild / Amy Reeve)
- [ ] Japanese text changes are proofread and approved (Junko Oda)
- [ ] UI changes look good in all themes (Alexander Rukin)
- [ ] Storybook works and no stories are broken (`yarn storybook`)
- [ ] In case of dependency changes `yarn.lock` file is updated

### Code Quality
- [ ] Important parts of the code are properly commented and documented
- [ ] Code is properly typed with flow
- [ ] React components are split-up enough to avoid unnecessary re-renderings
- [ ] Any code that only works in main process is neatly separated from components

### Testing
- [ ] New feature/change is covered by acceptance tests
- [ ] New feature/change is manually tested and approved by QA team
- [ ] All existing acceptance tests are still up-to-date
- [ ] New feature/change is covered by Daedalus Testing scenario
- [ ] All existing Daedalus Testing scenarios are still up-to-date

### After Review
- [ ] Merge the PR
- [ ] Delete the source branch
- [ ] Move the ticket to `done` column on the YouTrack board
- [ ] Update Slack QA thread by marking it with a green checkmark

### CODE REVIEW

**Comment by darko-mijic** [comment]
This:
image

should have the same color as the button:
image

@a-rukin, @yakovkaravelov, please fix.

**Comment by a-rukin** [comment]
> INCENTIVIZED TESTNET
> BALANCE CHECK
> here:
> image
> 
> should have the same color as the button:
> image
> 
> @a-rukin, @yakovkaravelov, please fix.

Updated zeplin links to avoid confusion...

File: source/renderer/app/components/loading/syncing-connecting/SyncingConnectingBackground.js [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      isConnecting,
+      isSyncing,
+      hasLoadedCurrentTheme,
+      isIncentivizedTestnet,
+    } = this.props;
+    const componentStyles = classNames([
+      styles.component,
+      hasLoadedCurrentTheme ? null : styles['is-loading-theme'],
+      isConnecting ? styles['is-connecting'] : null,
+      isSyncing ? styles['is-syncing'] : null,
```
Comment (tomothespian) [inline source/renderer/app/components/loading/syncing-connecting/SyncingConnectingBackground.js]: @yakovkaravelov can you please use camel case for classes

File: source/renderer/app/components/loading/syncing-connecting/SyncingConnectingBackground.scss [hunk 2]
```diff
@@ -0,0 +1,74 @@
+.component {
+  height: 100%;
+  left: 0;
+  object-fit: contain;
+  overflow: hidden;
+  position: absolute;
+  transition: background-color 500ms;
+  top: 0;
```
Comment (tomothespian) [inline source/renderer/app/components/loading/syncing-connecting/SyncingConnectingBackground.scss]: Change places of these 2 rules to match alphabetic order!

File: source/renderer/app/components/loading/syncing-connecting/SyncingConnectingTitle.scss [hunk 3]
```diff
@@ -1,32 +1,32 @@
...
+  .title {
+    color: var(--theme-syncing-connecting-title-color);
+    margin-bottom: 5px;
+    margin-top: 30px;
+  }
+  .subTitle1 {
+    color: var(--theme-syncing-connecting-subTitle1-color);
+    margin-bottom: 5px;
+  }
+  .subTitle2 {
```
Comment (daniloprates) [inline source/renderer/app/components/loading/syncing-connecting/SyncingConnectingTitle.scss]: Shouldn't this color be `#eb2256` instead?

image

**Comment by IuliaDolishniak** [comment]
@a-rukin @yakovkaravelov there is a different screen color at the bottom.
Screenshot from 2019-11-12 13-39-24

**Comment by a-rukin** [comment]
> @a-rukin @yakovkaravelov there is a different screen color at the bottom. I dont see anything like on Storybook or screens you attached to PR.
> Screenshot from 2019-11-12 13-39-24

Seems like gradient area is smaller than whole screen here

**Comment by a-rukin** [comment]
@nikolaglumac how can I help you?

**Comment by nikolaglumac** [comment]
@a-rukin get on a call with @yakovkaravelov and make sure the screen looks good 🙏

**Comment by yakovkaravelov** [comment]
@IuliaDolishniak @gnpf @a-rukin Everything was fixed.
You can retest now.
Thanks.

**Pull request merged**